os/binfmt/libelf/libelf_sections.c

```diff
@@ -65,13 +65,14 @@
 #include <tinyara/kmalloc.h>
 #include <tinyara/binfmt/elf.h>
 
-#include <queue.h>
 #include <string.h>
 #include <tinyara/binfmt/binfmt.h>
 #include <binary_manager/binary_manager.h>
 
 #include "libelf.h"
 
+static bin_addr_info_t g_bin_addr_list[CONFIG_NUM_APPS + 1];
```

Comment:
How about defining this only when loadable?

Reply:
Right. I added dependency of CONFIG_APP_BINARY_SEPARATION.